### PR TITLE
Webui 544

### LINKED ISSUE #544: WebUI - Allow to create new bug in the bug detail page
Sometimes when reading a timeline we detect is necessary to create other issues. So this feature will make it more agile than force user go back to the list page to create a new issue.

### PR DESCRIPTION
**Pull request by claudioantonio**
Fix #544

### CODE REVIEW

File: webui/src/components/BugTitleForm/BugTitleForm.tsx [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+interface Props {
+  bug: BugFragment;
+}
+
+/**
+ * Component for bug title change
+ * @param bug Selected bug in list page
+ */
+function BugTitleForm({ bug }: Props) {
+  const [bugTitleEditable, setBugTitleEditable] = useState(false);
```
Comment (MichaelMure) [inline webui/src/components/BugTitleForm/BugTitleForm.tsx:79]: maybe name that one `bugTitleEdition` ? "editable" makes me think about having the right to edit or not.

**Review by MichaelMure — APPROVED** [review]
Just some nitpicking about a naming. Looks good!

File: webui/src/components/BugTitleForm/SetTitle.graphql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+mutation setTitle($input: SetTitleInput!) {
+  setTitle(input: $input) {
+      bug {
+        humanId
```
Comment (MichaelMure) [inline webui/src/components/BugTitleForm/SetTitle.graphql:4]: you should retrieve the complete `id` here. `humanId` is a truncated Id, that's not enough to update the cache.

**Review by MichaelMure — CHANGES_REQUESTED** [review]
actually, there is this

File: webui/src/components/BugTitleForm/BugTitleForm.tsx (hunk 1 above)
Comment (claudioantonio) [reply webui/src/components/BugTitleForm/BugTitleForm.tsx:79]: I´ll change it.

File: webui/src/components/BugTitleForm/SetTitle.graphql (hunk 2 above)
Comment (claudioantonio) [reply webui/src/components/BugTitleForm/SetTitle.graphql:4]: Oh! I´ll fix it!

**Pull request merged**